.eslintrc.yml

```diff
@@ -1,5 +1,7 @@
 ---
-extends: eslint-config-shakacode/base
+# This needs to be commented to avoid conflict with the same config in the example app
+# extends: eslint-config-shakacode/base
+
```

Comment:
This part shouldn't be right, but I couldn't find any other way to make the eslint passes the test.